schema/paths/forms/types/form.yml

```
@@ -60,3 +60,12 @@ definitions:
         $ref: '#/components/schemas/form_id'
     required:
       - id
+
+parameters:
+  form_id:
+    name: formId
+    in: path
+    description: フォームID
+    required: true
+    schema:
+      $ref: '#/definitions/form_id'
```

このままだと、パスパラムに指定するスキーマはobjectではなく、integerやstringでないとうまくその値が展開されません
```suggestion
      $ref: '#/components/schemas/form_id'
```